cmd/konvoy-image-wrapper/cmd/wrapper.go

```diff
@@ -308,7 +315,14 @@ func (r *Runner) dockerRun(args []string) error {
 	}
 
 	for _, v := range r.volumes {
-		cmd.Args = append(cmd.Args, "--mount", "type="+v.kind+",source="+v.source+",target="+v.target)
+		bindOptions := ""
+		if len(v.bindOptions) > 0 {
+			bindOptions = fmt.Sprintf(",%s", strings.Join(v.bindOptions, ","))
+		}
+		cmd.Args = append(
+			cmd.Args,
+			"--mount",
+			"type="+v.kind+",source="+v.source+",target="+v.target+bindOptions)
 	}
```

Comment:
nit: refactoring this to a function would be nice for code readability

Reply:
I am not too sure about moving this to function. This function will be only called from this loop. If we have multiple callers for the same functionality we can think of factoring out it a function.